config/jobs/kubernetes/sig-node/dynamic-resource-allocation.yaml

```diff
@@ -98,6 +98,59 @@ periodics:
             cpu: 2
             memory: 9Gi
 
+  # This job runs the same tests as ci-node-e2e-crio-dra with some relevant feature gates (currently SchedulerQueueingHints) at non-default values.
+  - name: ci-node-e2e-crio-dra-features
```

Comment:
A pre-submit job seemed overkill for this, but at least having periodic checking should be done.